.github/workflows/release-merge.yaml

```diff
@@ -0,0 +1,39 @@
+name: Merge release changes to master
+on:
+  push:
+    branches:
+      - release-*
+env:
+  VERSION_FILE: pkg/version/release.go
+  DEFAULT_BRANCH: master
+jobs:
+  merge_release:
+    name: Merge release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Merge release branch into ${{env.DEFAULT_BRANCH}}
+        run: |
+          git checkout $DEFAULT_BRANCH
+          git checkout -b merge-${{ github.sha }}
+          # This info comes from https://api.github.com/users/github-actions%5Bbot%5D
+          # This way we make it look like github-actions[bot] made the commits
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git merge --no-commit --no-ff ${{ github.ref }}
```

Comment:
so it might be the case that we have multiple PRs to master and the later will contain changes of previous as well ?

Reply:
just want to clarify the flow, if there is the commit you don't want to merge to master, basically, as mentioned, you will close it. But then subsequent PR might also contain the changes of un-want commit.

Reply:
Yeah that's a good point. In that case we just have to rebase the branch ourselves from then on. But I think that should really be quite the exception, but maybe I'm missing a scenario

Reply:
@michaelbeaumont I am not sure either :sob:, sorry to bringing up subject which I don't fully understand.

One related note on releasing process (not related to this PR) is that I haven't seen any the patch fix (e.g. 0.22.1). Do we have supporting EOL policy as well (e.g. last 2 releases only or something like that)? One common approach is _backporting_, all commits will go to _master_, PR can be tagged with label for automating backport to specific release branch. There are plenty of available github actions and apps to support this workflow.

Reply:
Yeah, that's what I mentioned in the description. We don't really backport beyond the current RC/release at the moment. That's why this workflow works at all.

I personally prefer this since it keeps people up to date. I think it's something we have to consider when/if we introduce breaking changes.